build.fsx

```diff
@@ -45,8 +47,11 @@ let Run workingDirectory fileName args =
 
     ProcessResult.New code messages errors
 
-let dotnetHome = "C:\\Program Files\\dotnet\\"
+let dotnetHome = ".\\tools\dotnet\\"
 let dotnetExe = dotnetHome + "dotnet.exe"
+let dotnetInstall = dotnetHome + "dotnet-install.ps1"
+let dotnetInstallPath = "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1"
```

Comment:
Downloading it this way keeps the repository smaller. If builds take too long, we can just put it in the directory.